bitnami/mongodb/values.yaml

```
@@ -1002,6 +1002,57 @@ externalAccess:
       ##
       sessionAffinityConfig: {}
 
+## @section Network policy parameters
+##
+
+## Enable Network policies using Kubernetes Network Policies
+## ref: https://kubernetes.io/docs/concepts/services-networking/network-policies/
+##
+networkPolicy:
+  ## @param networkPolicy.enabled Enable MongoDB(&reg;) network policies
+  ##
+  enabled: false
+
+  ## Ingress rules - Allow incoming connections
+  ##
+  ingress:
+    ## @param networkPolicy.ingress.namespaceSelector [object] Namespace selector label that is allowed to access the MongoDB node. This label will be used to identify the allowed namespace(s).
+    ## @param networkPolicy.ingress.podSelector [object] Pod selector label that is allowed to access the MongoDB node. This label will be used to identify the allowed pod(s).
+    ## @param networkPolicy.ingress.customRules [array] Custom network policy for the MongoDB node.
```

Can you update the application name references as you did for `networkPolicy.enabled`?

```suggestion
    ## @param networkPolicy.ingress.namespaceSelector [object] Namespace selector label that is allowed to access the MongoDB(&reg;) node. This label will be used to identify the allowed namespace(s).
    ## @param networkPolicy.ingress.podSelector [object] Pod selector label that is allowed to access the MongoDB(&reg;) node. This label will be used to identify the allowed pod(s).
    ## @param networkPolicy.ingress.customRules [array] Custom network policy for the MongoDB(&reg;) node.
```

---

I think this suggestion was missed. Does it make sense to apply it?